lein-kibit/src/leiningen/kibit.clj

```diff
@@ -11,21 +11,20 @@
   (let [src-paths     (get-in project [:kibit :source-paths] ["rules"])
         repositories (:repositories project)
         local-repo    (:local-repo project)
-        kibit-project `{:dependencies [[jonase/kibit ~(str/trim-newline
-                                                        (slurp
-                                                          (io/resource
-                                                            "jonase/kibit/VERSION")))]]
+        kibit-version       (clojure.edn/read-string
+                             (slurp (io/resource "version.edn")))
+        kibit-project `{:dependencies [[gorillalabs/kibit ~(:version kibit-version)]]
```

Comment:
Would be neat if lein-v could also print out the group and artifact ID, so it didn't need to be hard-coded here.